src/radical/pilot/pmgr/launching/base.py

```diff
@@ -987,7 +980,7 @@ def _prepare_pilot(self, resource, rcfg, pilot, expand, tar_name):
         jd_dict.queue                 = queue
         jd_dict.candidate_hosts       = candidate_hosts
         jd_dict.file_transfer         = list()
-        jd_dict.environment           = dict()
+        jd_dict.environment           = {'RADICAL_SMT': smt}
```

Comment:
can we move adding this env into line 991? - to keep env setup together

```python
jd_dict.environment['RADICAL_BASE'] = resource_sandbox
jd_dict.environment['RADICAL_SMT']  = smt
```

Reply:
Done.